repos/system_upgrade/el7toel8/actors/authselectscanner/libraries/library.py

```diff
@@ -2,6 +2,7 @@
 import re
 import textwrap
 
+from six import PY2, PY3
```

Comment:
import PY2? Is it used anywhere?

Honestly I'd prefer `import six` and `six.PY3` usage, as it makes it instantly impossible to doubt where did PY2\PY3 come from.